raw/theme/html/html.css

```diff
@@ -824,40 +828,37 @@ a.co img { padding: 0; }
 
 /* ----------------- html toc ----------------- */
 
-div.toc ol {
-  margin-top: 8px !important;
-  margin-bottom: 8px !important;
-  margin-left: 0px !important;
-  padding-left: 0px !important;
-}
-
-div.toc ol ol {
-  margin-left: 30px !important;
-  padding-left: 0px !important;
+nav[data-type="toc"] li {
+  border: 0px solid #BEBEBE;  /* For debugging, set border non-zero. */
```

Comment:
Remove before merging? Not sure of "debugging" status in the comment

Reply:
Sure. The goal was to leave a _disabled_ styling, but maybe we'll never need that.

Reply:
Done!